src/Xibo/Support/Sanitizer/RespectSanitizer.php

```diff
@@ -261,6 +261,14 @@ public function getCheckbox($key)
         $value = $this->collection->get($key);
 
         // Validate the parameter
-        return ($value === 'on' || $value === 1 || $value === '1' || $value === 'true' || $value === true);
+        return ($value === 'on' || $value === 1 || $value === '1' || $value === 'true' || $value === true) ? 1 : 0;
```

Comment:
We need to make this change optional to support applications which have been written to expect true/false.